opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/consent/AisConsentInitiateBody.java

```diff
@@ -97,23 +101,23 @@ default Consents map(Xs2aAisContext cons) {
         }
 
         default AccountAccess.AvailableAccountsEnum accounts(String availableAccounts) {
-            if ("ALL_ACCOUNTS".equals(availableAccounts)) {
+            if (ALL_ACCOUNTS.getDescription().equals(availableAccounts)) {
                 return AccountAccess.AvailableAccountsEnum.ALLACCOUNTS;
             }
 
             return null;
         }
 
         default AccountAccess.AvailableAccountsWithBalance accountsWithBalance(String availableAccounts) {
-            if ("ALL_ACCOUNTS_WITH_BALANCES".equals(availableAccounts)) {
+            if (ALL_ACCOUNTS_WITH_BALANCES.getDescription().equals(availableAccounts)) {
```

Comment:
Suggest to use more meaningful name - 
getApiName (or apiName() or getName but it slightly clashes) instead of getDescription

Reply:
method was renamed to the getApiName